packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -57,19 +57,17 @@ export const RootScreen = ({ isReadyToSetupBackend }: RootScreenProps) => {
     () => dispatch(enterBackground())
   )
 
+  const canDismissSplashScreen =
```

Comment:
Maybe name this like `accountFetchResolved` or add a comment. It took me a bit to figure out that the status was of the fetch request not the account's status (deactivated, etc.)